addons/website/static/src/client_actions/configurator/configurator.scss

```diff
@@ -124,8 +123,9 @@
                         color: inherit;
                         outline: none;
                     }
+
                     &::placeholder {
-                        font-size: $input-font-size;
+                        font-weight: 300;
```

Comment:
If we want to adapt the placeholder's font properties, shouldn't that part be done in 17.0 when the placeholder appeared ?

Reply:
By the way, when the window is small, the field is not correctly aligned vertically - maybe this can be fixed as well.
![image](https://github.com/odoo/odoo/assets/71644421/620f89c7-190b-4b0b-9a91-925709b53167)

Reply:
Yes, But for every new improvement, the targeted version is the upcoming version.

For small device, Is it perfect?
![image](https://github.com/odoo/odoo/assets/99001002/32eed615-cea1-4d35-905e-127d71c6aaa6)

Reply:
When I draw an horizontal line, it seems the bottom of the "g" is visible while the bottom of the "y" is not, so I guess it is not properly aligned.
![image](https://github.com/odoo/odoo/assets/71644421/10699518-0d32-4f20-bd7c-6ee2cf031fd9)

Reply:
No reason to adapt this here indeed, either you do nothing (as the task is mentioning, this point is not to be done as it's already done..), or as BSO said, you can do it in 17.0.

I guess making it more thin/light is something we could do, but:
1. The task is mentioning it was already handled and not to be done anymore
2. Do it in 17
3. Keep the 40% size which was brainstormed and doesn't seem to have a reason to remove :+1:

Reply:
@bso-odoo changes are done :+1:

Reply:
@desh-odoo Maybe I am missing something, but based on the [message above](https://github.com/odoo/odoo/pull/147756/files#r1555936830), the fix about the industry field should be inside its own PR that starts at most in 17.0, and point (3) should be taken into account too: nothing specifies that the placeholder's style should change (even if I prefer your version).

Reply:
@mog-odoo discussed with @rdeodoo for point 2 and 3, to keep changes in this version. Can you please verify?

Reply:
Please, can we stop wasting time here?
**NO**, we did not discuss about keeping it here, I explained to MOG why we want it in 17, after already explaining that to you.

> **RDE**: I don't understand what is being discussed here, we know all that
We said to do it in 17🤷‍♂️ .
And the current PR is breaking the placeholder anyway
It just need to keep this part in Odoo 17 since we already introduce the placeholder there. Not much else to say
> **MOG**: Is it availble in the ucoming version using forward port?
> **RDE**: Do you mean, what will be done in 17 will also be done in master through the forward port ? Of course
> **MOG**: Yes
> **RDE**: Everything we do in stable ends up in master
> **MOG**: Okay thank for the clarification
> **RDE**: No problem, generally when we do a fix or a tiny visual improvement like this, we do it in stable, so it's helpful to people in stable, when the risk is close to 0
Here, if we judge the placeholder would be better with another style, and since it was introduced recently in 17, there is no reason to not do it in 17
But it would be ok to do it in master only, it wouldn't be something too bad
It's just that we might as well benefit from the visual for "one year of Odoo version"